Gaijinizer patch/Map037.txt

```diff
@@ -1076,28 +1103,29 @@ with you in the previous passage.#MTLed
 優しく、慈しみを込めて\\N[1]様を堕とそうとし
 与えた余裕すら最後に潰し、抵抗の希望を奪いました。
 > CONTEXT: Map037/events/4/pages/1/501/Dialogue/yuuma
-At that time, I was convinced to win.
-Gently, with kindness, I try to jealousy \\N[1]
-I finally smashed even the room I gave it and
-took away the hope of the struggle#MTLed
+At the time, I was convinced I'd win.
+I tried to corrupt you with love and kindness,
+\\N[1]. Even with all the leeway you had, I was
+going to steal away any hope of resistance from
+you.#Ron
 > END STRING
 
 > BEGIN STRING
 しかし……貴方は抵抗しました。
 
 無理だとわかっているのに、無駄だとわかっているのに…。
 > CONTEXT: Map037/events/4/pages/1/506/Dialogue/yuuma
-But...you struggled. I know it is impossible,
-but I know it is useless ....#MTLed
+But... you struggled. Even though you knew it was
+impossible, even though you knew...#Ron
 > END STRING
 
 > BEGIN STRING
 あの時点で、私の…『アイアンメイデン』は負けたのです。
 
 貴方の『人としての意志』に。
 > CONTEXT: Map037/events/4/pages/1/510/Dialogue/yuuma
-At that point, my..."Iron Maiden" lost.
-In your 'will as a person'.#MTLed
+At that point, my "Iron Maiden" was beaten by your
+"will as a person".#Ron
 > END STRING
```

Comment:
Given all the Undertale side references and inspiration in SHRIFT, it would be a shame not to translate "Will" as "Determination". :3 Let's try something like this:
>_**At that point, my will as an "Iron Maiden" lost...
...to your "Determination" as a human being.**_

(There's a lot of ways we can do this, feel free to throw any ideas at me.)

Reply:
Hmm, I hadn't thought of doing that. I'll keep an eye out for opportunities.

On that note, I just had a look at some Japanese Let's Plays, and the word used in the Japanese version of Undertale for "Determination" was 決意 so if you see that, it's probably the dev directly invoking Undertale in that scene.